src/pip/_internal/utils/misc.py

```diff
@@ -59,9 +59,9 @@
 except ImportError:
     # typing's cast() isn't supported in code comments, so we need to
     # define a dummy, no-op version.
-    def cast(typ, val):
+    def cast(typ, val):  # type: ignore
         return val
-    VersionInfo = None
+    VersionInfo = None  # type: ignore
```

Comment:
Same here.

Reply:
You could try using the `as` workaround used here in PR #6704: https://github.com/pypa/pip/pull/6704/files#diff-f31f8cfd2ff510432f665b48dfe46646R33